### PR TITLE
[1.13.x-blue] update brew packages name, repositories names and branches

### PR DESCRIPTION
**Pull request by davidesalerno**
Many thanks for submitting your Pull Request :heart:! 

Please make sure your PR meets the following requirements:

- [x] You have read the [contributors' guide](https://github.com/kiegroup/kogito-operator/blob/main/README.md#contributing-to-the-kogito-operator)
- [ ] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [ ] Pull Request contains a link to the JIRA issue
- [ ] Pull Request contains a description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket
- [x] Your feature/bug fix has a unit test that verifies it
- [x] You've ran `make before-pr` and everything is working accordingly
- [ ] You've tested the new feature/bug fix in an actual OpenShift cluster
- [ ] You've added a [RELEASE_NOTES.md](https://github.com/kiegroup/kogito-operator/blob/main/RELEASE_NOTES.md) entry regarding this change

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>Run operator BDD testing</b>
  Please add comment: <b>/jenkins test</b>

* <b>Run RHPAM operator BDD testing</b>
  Please add comment: <b>/jenkins RHPAM test</b>
</details>

### CODE REVIEW

**Review by ricardozanini — APPROVED** [review]

File: rhpam-image-bundle.yaml [hunk 1]
```diff
@@ -65,5 +65,5 @@ osbs:
           - ppc64le
   extra_dir: osbs-extra
   repository:
-    name: containers/rhpam-7-kogito-operator-bundle
-    branch: rhba-7-rhel-8
+    name: containers/bamoe-kogito-rhel8-operator-bundle
```
Comment (rgdoliveira) [inline rhpam-image-bundle.yaml:68]: it should be ibm-bamoe-kogito-operator-bundle

File: rhpam-image-prod.yaml [hunk 2]
```diff
@@ -55,8 +55,8 @@
   osbs:
     extra_dir: osbs-extra
     repository:
-      name: containers/rhpam-7-kogito-operator
-      branch: rhba-7-rhel-8
+      name: containers/bamoe-kogito-rhel8-operator
```
Comment (rgdoliveira) [inline rhpam-image-prod.yaml:58]: it should be ibm-bamoe-kogito-operator

File: rhpam-image.yaml [hunk 3]
```diff
@@ -31,7 +31,7 @@
     - name: maintainer
       value: bsig-cloud@redhat.com
     - name: com.redhat.component
-      value: rhpam-7-kogito-rhel8-operator-container
+      value: bamoe-kogito-rhel8-operator-container
```
Comment (rgdoliveira) [inline rhpam-image.yaml:34]: it should be ibm-bamoe-kogito-rhel8-operator-container

File: rhpam-image.yaml [hunk 4]
```diff
@@ -57,8 +57,8 @@
             - ppc64le
     extra_dir: osbs-extra
     repository:
-      name: containers/rhpam-7-kogito-operator
-      branch: rhba-7-rhel-8
+      name: containers/bamoe-kogito-rhel8-operator
```
Comment (rgdoliveira) [inline rhpam-image.yaml:60]: it should be ibm-bamoe-kogito-operator

**Review by rgdoliveira — CHANGES_REQUESTED** [review]

**Review by rgdoliveira — APPROVED** [review]
LGTM

**Review by spolti — APPROVED** [review]

**Pull request merged**